common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -0,0 +1,752 @@
+        self.answer_problem(False)
+        self.problem_page.click_check()
+        self.wait_for_status('incorrect')
+
+    @attr('shard_2')
+    def test_submit_blank_answer(self):
+        """
+        Scenario: I can submit a blank answer
+        Given I am viewing a "<ProblemType>" problem
+        When I check a problem
+        Then my "<ProblemType>" answer is marked "incorrect"
+        And The "<ProblemType>" problem displays a "blank" answer
+        """
+        # Make sure we're looking at the right problem
+        self.assertEqual(self.problem_page.problem_name, self.problem_name)
+
+        # Leave the problem unchanged and click check.
+        self.problem_page.click_check()
+        self.wait_for_status('incorrect')
+
+    @attr('a11y')
+    def test_problem_type_a11y(self):
+        """
+        Run accessibility audit for the problem type.
+        """
+        # Make sure we're looking at the right problem
+        self.assertEqual(self.problem_page.problem_name, self.problem_name)
+
+        # Set the scope to the problem container
+        self.problem_page.a11y_audit.config.set_scope(
+            include=['div#seq_content'])
+
+        # Run the accessibility audit.
+        self.problem_page.a11y_audit.check_for_accessibility_errors()
+
+
+class CheckboxProblemTypeTest(ProblemTypeTestBase, ProblemTypeTestMixin):
+    """
+    TestCase Class for Checkbox Problem Type
+    """
+    problem_name = 'CHECKBOX TEST PROBLEM'
+    problem_type = 'checkbox'
+
+    factory = ChoiceResponseXMLFactory()
+
+    factory_kwargs = {
+        'question_text': 'The correct answer is Choice 3',
+        'choice_type': 'checkbox',
+        'choices': [True, False, True, False],
+        'choice_names': ['Choice 0', 'Choice 1', 'Choice 2', 'Choice 3']
+    }
+
+    def setUp(self, *args, **kwargs):
+        """
+        Additional setup for CheckboxProblemTypeTest
+        """
+        super(CheckboxProblemTypeTest, self).setUp(*args, **kwargs)
+        self.problem_page.a11y_audit.config.set_rules({
+            'ignore': [
+                'aria-allowed-attr',  # TODO: AC-251
+                'aria-valid-attr',  # TODO: AC-251
+                'aria-roles',  # TODO: AC-251
+                'checkboxgroup',  # TODO: AC-251
+            ]
+        })
+
+    def answer_problem(self, correct):
+        """
+        Answer checkbox problem.
+        """
+        if correct:
+            self.problem_page.click_choice("choice_0")
+            self.problem_page.click_choice("choice_2")
+        else:
+            self.problem_page.click_choice("choice_1")
+
+
+class MultipleChoiceProblemTypeTest(ProblemTypeTestBase, ProblemTypeTestMixin):
+    """
+    TestCase Class for Multiple Choice Problem Type
+    """
+    problem_name = 'MULTIPLE CHOICE TEST PROBLEM'
+    problem_type = 'multiple choice'
+
+    factory = MultipleChoiceResponseXMLFactory()
+
+    factory_kwargs = {
+        'question_text': 'The correct answer is Choice 2',
```

Comment:
I think it's actually choice 3, since a visual look at the page indexes at 1 but python indexes at 0? That's what's throwing me off with the text, I guess.